corehq/apps/smsbillables/models.py

```diff
@@ -275,15 +281,18 @@ class Meta:
 
     @property
     def gateway_charge(self):
+        amount = None
         if self.gateway_fee is not None:
             try:
-                charge = SmsGatewayFee.objects.get(id=self.gateway_fee.id)
-                if self.gateway_fee_conversion_rate is not None:
-                    return charge.amount / self.gateway_fee_conversion_rate
-                return charge.amount
+                amount = SmsGatewayFee.objects.get(id=self.gateway_fee.id).amount
```

Comment:
I left this legacy setup as is; probably it can be simplified unless there is some concern about what would happen if a gateway fee is deleted from the database (however unlikely)